src/components/organisms/UiPhoneNumber/_internal/UiPhoneNumberPrefix.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+  <UiDropdown
+    class="ui-phone-number__dropdown"
+    :model-value="modelValue"
+    :popover-attrs="{ class: 'ui-phone-number__dropdown-popover' }"
+    @update:model-value="handleOnChangeCode"
+  >
+    <template
+      #toggle="{
+        toggleHandler, isOpen,
+      }"
+    >
+      <UiButton
+        id="ui-phone-number__dropdown"
```

Comment:
why do we need this ID? it can't be unique when user will use 2 UiPhoneNumbers on the page

Reply:
I set it for label which I added to UiPhoneNumber